student.rb

```diff
@@ -1,6 +1,8 @@
 require './person'
 
 class Student < Person
```

Comment:
- [ ] You have correctly set up the `has-many` end of the association in the Classroom class. However, your Student class should also have a method for the other side of association setup `(belongs-to)`. You can refer to this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for an example of how to set up the association both ways.